ITEvents/Screens/Search tab/SearchPresenter.swift

```diff
@@ -85,7 +85,7 @@ class SearchPresenter: ISearchPresenter {
             return
         }
         
-        loadBatchEvents()
+        _ = loadBatchEvents()
```

Comment:
We indeed need to add ability to cancel the initial request as well, if user start typing very fast in the search textbox once he tapped to "search" tab:
```swift
searchCancelation = loadBatchEvents()
```